pybamm/install_odes.py

```diff
@@ -8,28 +8,32 @@
 
 from pybamm.util import root_dir
 
-try:
-    # wget module is required to download SUNDIALS or SuiteSparse.
-    import wget
+def check_platform():
+    if sys.platform == "win32":
+        raise Exception("pybamm_install_odes is not supported on Windows.")
```

Comment:
Do we really need a separate function to do this considering that this file is a script meant to be executed rather than an importable file, and will therefore run from top to bottom?

Reply:
I've unwrapped it and eliminated the function. Thanks.